docs/USER-DOCUMENTATION.md

```diff
@@ -149,6 +149,14 @@ In Windows:
 set ETCHER_DISABLE_UPDATES=1
 ```
 
+Simulate an update alert
+------------------------
+
+You can set the `ETCHER_LATEST_VERSION` environment variable to a valid semver
```

Comment:
I wonder if we should make this variable slightly more specific e.g. `ETCHER_FAKE_LATEST_S3_VERSION`, since I guess `ETCHER_LATEST_VERSION` could mean many different things?